<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>833</width>
    <height>565</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="ledOneImage">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>20</y>
      <width>67</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resource.qrc">:/images/images/blub.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="ledOneCheckBox">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>40</y>
      <width>92</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>LED 1</string>
    </property>
    <property name="shortcut">
     <string>1</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="ledTwoCheckBox">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>140</y>
      <width>92</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>LED 2</string>
    </property>
    <property name="shortcut">
     <string>2</string>
    </property>
   </widget>
   <widget class="QLabel" name="ledTwoImage">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>110</y>
      <width>67</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resource.qrc">:/images/images/blub.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="ledThreeCheckBox">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>210</y>
      <width>92</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>LED 3</string>
    </property>
    <property name="shortcut">
     <string>3</string>
    </property>
   </widget>
   <widget class="QLabel" name="ledThreeImage">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>190</y>
      <width>67</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resource.qrc">:/images/images/blub.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="ledFourCheckBox">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>310</y>
      <width>92</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>LED 4</string>
    </property>
    <property name="shortcut">
     <string>4</string>
    </property>
   </widget>
   <widget class="QLabel" name="ledFourImage">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>280</y>
      <width>67</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resource.qrc">:/images/images/blub.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>60</y>
      <width>121</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Switching Speed</string>
    </property>
   </widget>
   <widget class="QPushButton" name="shineLedButton">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>400</y>
      <width>121</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>LED Shining</string>
    </property>
    <property name="shortcut">
     <string>S</string>
    </property>
   </widget>
   <widget class="QSlider" name="periodSlider">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>100</y>
      <width>160</width>
      <height>16</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="tracking">
     <bool>false</bool>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="periodSpinBox">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>150</y>
      <width>48</width>
      <height>26</height>
     </rect>
    </property>
    <property name="keyboardTracking">
     <bool>false</bool>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
   </widget>
   <widget class="QProgressBar" name="periodBar">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>150</y>
      <width>118</width>
      <height>23</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QPushButton" name="switchingOnButton">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>200</y>
      <width>141</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Switching on</string>
    </property>
    <property name="shortcut">
     <string>O</string>
    </property>
   </widget>
   <widget class="QPushButton" name="switchingOffButon">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>250</y>
      <width>141</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Switching off</string>
    </property>
    <property name="shortcut">
     <string>I</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>833</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>periodSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>periodBar</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>389</x>
     <y>146</y>
    </hint>
    <hint type="destinationlabel">
     <x>398</x>
     <y>204</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>periodSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>periodSpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>469</x>
     <y>152</y>
    </hint>
    <hint type="destinationlabel">
     <x>519</x>
     <y>213</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>periodBar</sender>
   <signal>valueChanged(int)</signal>
   <receiver>periodSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>347</x>
     <y>200</y>
    </hint>
    <hint type="destinationlabel">
     <x>345</x>
     <y>143</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>periodBar</sender>
   <signal>valueChanged(int)</signal>
   <receiver>periodSpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>419</x>
     <y>206</y>
    </hint>
    <hint type="destinationlabel">
     <x>502</x>
     <y>206</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>periodSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>periodBar</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>485</x>
     <y>192</y>
    </hint>
    <hint type="destinationlabel">
     <x>440</x>
     <y>193</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>periodSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>periodSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>502</x>
     <y>198</y>
    </hint>
    <hint type="destinationlabel">
     <x>479</x>
     <y>145</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
